drivers/include/pca9633.h

```diff
@@ -30,39 +30,10 @@ extern "C"
 
 #include "pca9633_regs.h"
 
-/* Frequency of 24 Hz is used */
-/**
- * @brief Blinking period with a duration of 125 ms
- *
- * ((1 / 24 Hz) * 3 cycles)
- */
-#define PCA9633_BLINKING_PERIOD_125_MS  3
-
-/**
- * @brief Blinking period with a duration of 250 ms
- *
- * ((1 / 24 Hz) * 6 cycles)
- */
-#define PCA9633_BLINKING_PERIOD_250_MS  6
-
-/**
- * @brief Blinking period with a duration of 500 ms
- *
- * ((1 / 24 Hz) * 12 cycles)
- */
-#define PCA9633_BLINKING_PERIOD_500_MS  12
-
-/**
- * @brief Blinking period with a duration of 1 s
- *
- * ((1 / 24 Hz) * 24 cycles)
- */
-#define PCA9633_BLINKING_PERIOD_1_S     24
-
 /**
  * @brief Blinking period with a maximum duration of ~10.73 s
  */
-#define PCA9633_BLINKING_PERIOD_MAX     255
+#define PCA9633_BLINKING_PERIOD_MAX     (65535)
```

Comment:
Shouldn't that be 10625?

Reply:
It doesn't matter. I simply took the largest value possible such that the if-condition in
```
uint16_t blink_period = (24 * blink_period_ms) / 1000;
if (blink_period > 255) {
    blink_period = 255;
}
```
is triggered and the value will be capped to 255 for the register value anyway.

Reply:
But if you think its confusing I can change it.

Reply:
It *is* confusing.
Since this is not used anywhere, it will only lead unsuspecting users astray ;)

If you change it, also better change it to `PCA9633_BLINKING_PERIOD_MAX_MS` so it's apparent what unit that number is in.

Reply:
ok^^ I changed it and I also mentioned the define in the function's documentation now.